packages/api/docs/labels.md

```diff
@@ -0,0 +1,826 @@
+<!-- this doc is generated by ./scripts/docs/labels.mjs -->
+
+  # Labels
+  
+  This document is a reference for the labels used in the SDK.
+
+  **⚠️ Note**: These labels are still in development and may change over time. Not all are currently in use.
+
+  ## Key
+
+  ### Label Preferences
+
+  The possible client interpretations for a label.
+
+  - <code>ignore</code> Do nothing with the label.
+  - <code>warn</code> Provide some form of warning on the content (see "On Warn" behavior).
+  - <code>hide</code> Remove the content from feeds and apply the warning when directly viewed.
+
+  Each label specifies which preferences it can support. If a label is not configurable, it must have only own supported preference.
+
+  ### Configurable?
+
+  Non-configurable labels cannot have their preference changed by the user.
+
+  ### Flags
+
+  Additional behaviors which a label can adopt.
+
+  - <code>no-override</code> The user cannot click through any covering of content created by the label.
+  - <code>adult</code> The user must have adult content enabled to configure the label. If adult content is not enabled, the label must adopt the strictest preference.
```

Comment:
Not blocking, and might not be worth changing at this point, but we might want to use a different keyword here to distinguish between "account is/isn't in show-adult-content mode" from "account is an adult". It is at least conceivable that this SDK will specify different behaviors for minors vs. for adults, in a way that can't be overridden by configuration.